tests/system/test_integration_acm.py

```diff
@@ -0,0 +1,247 @@
+import time
+import unittest
+from urlparse import urljoin
+import uuid
+
+import requests
+
+from apmserver import ElasticTest
+from beat.beat import INTEGRATION_TESTS
+
+
+class AgentConfigurationIntegrationTest(ElasticTest):
+    config_overrides = {
+        "logging_json": "true",
+        "kibana_enabled": "true",
+        "acm_cache_expiration": "1ms",  # no cache
```

Comment:
sidenote: This won't work with https://github.com/elastic/apm-server/pull/2443. But let's first merge in this PR and worry about that later.